src/components/Dashboard/FirstSteps/reducers/firstStepsReducer.js

```diff
@@ -1,7 +1,11 @@
+import { mapSystemUsageSummary } from '..';
+
 export const PENDING_STATUS = 0;
 export const COMPLETED_STATUS = 1;
 export const WARNING_STATUS = 2;
+export const UNKNOWN_STATUS = 3;
```

Comment:
What do you think about using names in place of integers?

Reply:
I don't have a strong opinion about this ...